test/basic/ecs_web_update/ecs_web_update_test.go

```diff
@@ -0,0 +1,707 @@
+		assert.EqualValues(t, ecsRevertLbGet.StatusCode, 200)
+	}
+
+	ecsRevertServiceUrl := terraform.Output(t, terraformRevertOptions, "web_url")
+	ecsRevertUrlGet, ecsRevertUrlGetErr := http.Get(fmt.Sprintf("https://%s", ecsRevertServiceUrl))
+	if assert.NoError(t, ecsRevertUrlGetErr) {
+		assert.EqualValues(t, ecsRevertUrlGet.StatusCode, 200)
+	}
+}
+
+func TestECSWebChangeContainer(t *testing.T) {
+	initialVariables := map[string]interface{}{
+		"organization_id":   os.Getenv("ORGANIZATION_ID"),
+		"environment_id":    os.Getenv("ENVIRONMENT_ID"),
+		"aptible_host":      os.Getenv("APTIBLE_HOST"),
+		"dns_account_id":    os.Getenv("DNS_AWS_ACCOUNT_ID"),
+		"ecs_name":          "ecs-pub-web-test-container",
+		"container_command": []string{"nginx", "-g", "daemon off;"},
+		"container_image":   "nginx",
+		"container_port":    80,
+		"container_name":    "nginx",
+		"is_public":         true,
+		"is_ecr_image":      false,
+		"vpc_name":          "testecs-container-vpc",
+		"domain":            "aptible-cloud-staging.com",
+		"subdomain":         "test-ecs-integration",
+	}
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: ".",
+		Vars:         initialVariables,
+	})
+
+	defer cleanupAndAssert(t, terraformOptions)
+	terraform.InitAndApply(t, terraformOptions)
+
+	c := client.NewClient(
+		true,
+		os.Getenv("APTIBLE_HOST"),
+		os.Getenv("APTIBLE_TOKEN"),
+	)
+	ctx := context.Background()
+
+	aptibleAccountId := terraform.Output(t, terraformOptions, "aptible_aws_account_id")
+	aptibleAccountRole := fmt.Sprintf("arn:aws:iam::%s:role/OrganizationAccountAccessRole", aptibleAccountId)
+	os.Setenv(terratest_aws.AuthAssumeRoleEnvVar, aptibleAccountRole)
+
+	vpcId := terraform.Output(t, terraformOptions, "vpc_id")
+	vpcAsset, vpcAws, vpcErr := getAptibleAndAWSVPCs(t, ctx, c, vpcId, "testecs-container-vpc")
+	if assert.NoError(t, vpcErr) {
+		assertCommonVpc(t, vpcId, vpcAsset, vpcAws)
+	}
+
+	ecsWebId := terraform.Output(t, terraformOptions, "ecs_web_id")
+	ecsWebAsset, ecsClusterAws, ecsServiceAws, ecsErr := getAptibleAndAWSECSServiceAndCluster(t, ctx, c, ecsWebId, "ecs-pub-web-test-container-web-cluster", "ecs-pub-web-test-container")
+	if assert.NoError(t, ecsErr) {
+		assertCommonEcs(t, ecsWebId, ecsWebAsset, ecsClusterAws, ecsServiceAws)
+	}
+	assert.Equal(t, int64(1), *ecsServiceAws.DesiredCount)
+	taskDefinition := terratest_aws.GetEcsTaskDefinition(t, "us-east-1", *ecsServiceAws.TaskDefinition)
+	assert.Equal(t, "awsvpc", *taskDefinition.NetworkMode)
+	assert.Equal(t, 2, len(taskDefinition.ContainerDefinitions)) // ssl proxy and service container
+	var nginxFound bool
+	for _, container := range taskDefinition.ContainerDefinitions {
+		if *container.Name == "nginx" {
+			assert.Equal(t, "nginx", *container.Image)
+			assert.Equal(t, []*string{
+				aws.String("nginx"),
+				aws.String("-g"),
+				aws.String("daemon off;"),
+			}, container.Command)
+		}
+	}
+	assert.True(t, nginxFound)
```

Comment:
how does this pass 🤔

Reply:
this teaches me to not move around code without running the tests even if they take 30+ mins 😮‍💨